backup/setup/instance-configs.json

```diff
@@ -0,0 +1,26 @@
+[
+	{
+		"_id": {
+			"$oid": "666076ad83d1e69b5c692efd"
+		},
+		"name": "nbc"
+	},
+	{
+		"_id": {
+			"$oid": "666076b192e0f29a612935d1"
+		},
+		"name": "thr"
+	},
+	{
+		"_id": {
+			"$oid": "666076b4ccb83d0360de2802"
+		},
+		"name": "brb"
+	},
+	{
+		"_id": {
+			"$oid": "666076b859c84c275ec840c2"
+		},
+		"name": "default"
```

Comment:
```suggestion
		"name": "dbc"
```

Reply:
i thing `default` is fine, for all other instances who are not `nbc`, `thr` and `brb`.
As example Audit or the infra instances

Reply:
@mamutmk5 
@CeEv said that would be correct. It is name of instance and not theme

Reply:
@SevenWaysDPthen @CeEv we need much more than this 4 in it and the Instance name is more then this.
This the things are Internal short names.
Then stop to try the make from the Thema to the instnace name.
And the name default is like an switch case thing, with the default case if no other case fits.